src/components/Resources/FilterTags.tsx

```diff
@@ -0,0 +1,43 @@
+import client from '@/database/client.tsx'
```

Comment:
I really like the functionality but I worry it's not scalable. Would like to iterate on this.